### DISCUSSION TITLE
Komma Trennung in Zahl

### DISCUSSION THREAD

**Discussion by SaschaH88** [jomjol/AI-on-the-edge-device#1758] [General] [answered]
Hallo zusammen 

Habe einen Zahler ohne Analogen Nadeln. Soweit hinbekommen ! Aber wie setzt ich nun das Komma an der richtigen Stelle ?
![B9B38709-BD3B-46E7-975B-3159467D3A9E](https://user-images.githubusercontent.com/120577258/210447670-e9e62148-0a47-40e3-809f-2f9c5a10e13a.png)

**Comment by jurasjo** [comment, marked as answer]
Settings->Configuration
![Zrzut ekranu z 2023-01-04 07-30-29](https://user-images.githubusercontent.com/19259405/210490389-0c18b843-7a88-42df-b35a-6528953e35d4.png)
